src/wl_window.c

```diff
@@ -429,14 +454,14 @@ void _glfwPlatformSetWindowSizeLimits(_GLFWwindow* window,
                                       int minwidth, int minheight,
                                       int maxwidth, int maxheight)
 {
-    // TODO
-    fprintf(stderr, "_glfwPlatformSetWindowSizeLimits not implemented yet\n");
+    // TODO: find out how to trigger a resize.
```

Comment:
Do you mean interactive resize here? What we need to do is to add client side window decorations where we would trigger resize and move. How exactly to do these I'd have to ask @elmindreda for her opinion. Initially we could just add a few transparent pixels around (using 4 subsurfaces), but might want something better than that later.